src/GraphQl/Resolver/ResourceAccessCheckerTrait.php

```diff
@@ -30,17 +30,24 @@ trait ResourceAccessCheckerTrait
     /**
      * @throws Error
      */
-    public function canAccess(?ResourceAccessCheckerInterface $resourceAccessChecker, ResourceMetadata $resourceMetadata, string $resourceClass, ResolveInfo $info, $extraVariables = [], string $operationName = null): void
+    public function canAccess(?ResourceAccessCheckerInterface $resourceAccessChecker, ResourceMetadata $resourceMetadata, string $resourceClass, ResolveInfo $info, $extraVariables = [], string $operationName = null, string $attribute = 'security'): void
     {
         if (null === $resourceAccessChecker) {
             return;
         }
 
-        $isGranted = $resourceMetadata->getGraphqlAttribute($operationName ?? '', 'access_control', null, true);
+        $isGranted = $resourceMetadata->getGraphqlAttribute($operationName ?? '', $attribute, null, true);
+        if (null === $isGranted) {
+            $isGranted = $resourceMetadata->getGraphqlAttribute($operationName ?? '', 'access_control', null, true);
```

Comment:
As this class is experimental, you could also just make the `$attribute` mandatory, and trigger the deprecation only if its value is `access_control`